[bug] react 19, zustand 5.02 버전 충돌

``` diff
- npm error code ERESOLVE
- npm error ERESOLVE could not resolve
- npm error
- npm error While resolving: zustand@5.0.2
- npm error Found: react@19.0.0-rc-66855b96-20241106
- npm error node_modules/react
- npm error   react@"19.0.0-rc-66855b96-20241106" from the root project
- npm error   peer react@"^18.2.0 || 19.0.0-rc-66855b96-20241106" from next@15.0.3
- npm error   node_modules/next
- npm error     next@"15.0.3" from the root project
- npm error   3 more (next-themes, react-dom, styled-jsx)
```

``` diff
- npm error code ERESOLVE
- npm error ERESOLVE unable to resolve dependency tree
- npm error
- npm error While resolving: didit@0.1.0
- npm error Found: react@19.1.0-canary-fc8a898d-20241226
- npm error node_modules/react
- npm error   react@"^19.1.0-canary-056073de-20250109" from the root project
- npm error
- npm error Could not resolve dependency:
- npm error peer react@"^18.2.0 || 19.0.0-rc-de68d2f4-20241204 || ^19.0.0" from next@15.1.4
- npm error node_modules/next
- npm error   next@"15.1.4" from the root project
- npm error
- npm error Fix the upstream dependency conflict, or retry
- npm error this command with --force or --legacy-peer-deps
- npm error to accept an incorrect (and potentially broken) dependency resolution.
- npm error
- npm error
- npm error For a full report see:
- npm error /Users/muzi5/.npm/_logs/2025-01-11T07_42_21_477Z-eresolve-report.txt
- npm error A complete log of this run can be found in: /Users/muzi5/.npm/_logs/2025-01-11T07_42_21_477Z-debug-0.log
```

react 버전을 바꾸면 Next와 버전충돌이 일어남

--- a/package.json
+++ b/package.json
@@ -11,8 +11,8 @@
   "dependencies": {
     "next": "15.0.3",
     "next-themes": "^0.4.4",
-    "react": "^19.1.0-canary-056073de-20250109",
-    "react-dom": "^19.1.0-canary-056073de-20250109"
+    "react": "19.0.0-rc-66855b96-20241106",
+    "react-dom": "19.0.0-rc-66855b96-20241106"
   },
   "devDependencies": {
     "@types/node": "^20",

